charts/jaeger/templates/allinone-agent-svc.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ template "jaeger.agent.name" . }}
+  name: {{ include "jaeger.fullname" . }}-agent
```

Comment:
what is the role of this file?

allinone-agent-svc.yaml

If it's still needed please add a header comment

Reply:
it looks like it creates a "service" for UDP ports, but why do we need a dedicated name for it like "xx-agent" and "xx-collector" below? These terms are now mostly meaningless in Jaeger v2 architecture, every port that receives the data is a "collector" function.